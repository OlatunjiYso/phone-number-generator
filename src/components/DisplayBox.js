import React from 'react';

function DisplayBox(props) {
  const { content } = props;
  return (
    <div className="display-box">
      <h2> {content} </h2>
    </div>
  );
}

export default DisplayBox;
