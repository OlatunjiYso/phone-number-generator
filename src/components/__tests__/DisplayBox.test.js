import React from 'react';
import { shallow } from 'enzyme';
import DisplayBox from '../DisplayBox';

it('renders the DisplayBox component', () => {
  const wrapper = shallow(<DisplayBox />);
  expect(wrapper).toMatchSnapshot()
});