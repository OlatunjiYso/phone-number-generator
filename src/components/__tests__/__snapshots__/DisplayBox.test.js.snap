// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the  component 1`] = `ShallowWrapper {}`;

exports[`renders the App component 1`] = `ShallowWrapper {}`;

exports[`renders the DisplayBocx component 1`] = `ShallowWrapper {}`;

exports[`renders the DisplayBox component 1`] = `ShallowWrapper {}`;
